packages/readabilityjs/Readability.js

```diff
@@ -168,8 +168,8 @@ Readability.prototype = {
     // NOTE: These two regular expressions are duplicated in
     // Readability-readerable.js. Please keep both copies in sync.
     articleNegativeLookBehindCandidates: /breadcrumbs|breadcrumb|utils/i,
-    articleNegativeLookAheadCandidates: /outstream(.?)_|sub(.?)_/i,
-    unlikelyCandidates: /-ad-|ai2html|banner|breadcrumbs|breadcrumb|combx|comment|community|cover-wrap|disqus|extra|footer|gdpr|header|legends|menu|related|remark|replies|rss|shoutbox|sidebar|skyscraper|social|sponsor|supplemental|ad-break|agegate|pagination|pager(?!ow)|popup|yom-remote|copyright|keywords|outline|infinite-list|beta|recirculation|site-index|hide-for-print|post-end-share-cta|post-end-cta-full|post-footer|main-navigation|programtic-ads|outstream_article|hfeed|comment-holder/i,
+    articleNegativeLookAheadCandidates: /outstream(.?)_|sub(.?)_|m_/i,
```

Comment:
Just wondering how this one shows up?

Reply:
The top and right arrow are images with class name `m_article_blahblahblah go_to_top/right` and it matches the `articleNegativeLookAheadCandidates` regex so it would not be removed as a `okMaybeItsACandidate` element.

That's why I added `m_` in the `articleNegativeLookAheadCandidates` and `back-to-top|show-up-next` to the `unlikelyCandidates` to remove these arrow images
@jacksonh